### PR TITLE
Prakash orderattachment download button

### PR DESCRIPTION
**Pull request by BorisovskiP**
Add missing download button

### CODE REVIEW

File: view/base/web/template/attachment-markup.html [hunk 1]
```diff
@@ -39,6 +39,7 @@ <h4 data-bind="i18n: 'Drag or Select your files here'"></h4>
                                                 class="comment" placeholder="Write comment here">
                                                 </textarea>
 
+                                                <a id="sp-attachment-download1" class="sp-attachment-download" title="Download Item" data-bind="click: $parent.downloadFile"  href="#" name="downloadItem"></a>
```
Comment (sprankhub) [inline view/base/web/template/attachment-markup.html]: Why did you add a `name` here? I think this is not needed?

Comment (sprankhub) [inline view/base/web/template/attachment-markup.html]: There is a duplicate white space before the `href` :smile:

**Review by sprankhub — APPROVED** [review]

**Comment by sprankhub** [comment]
Could you merge this, @prakash282?

**Comment by Elema82** [comment]
@sprankhub I just found this issue. Great update!

**Comment by sprankhub** [comment]
Good to hear you like it @Elema82 :) But there is still an issue, which needs to be fixed by @BorisovskiP :)

File: Helper/Attachment.php [hunk 2]
```diff
@@ -154,7 +154,7 @@ public function saveAttachment($request)
                     'hash' => $attachment->getHash()
                 ]
             );
-            $download = $defaultStore->getUrl(
+            $download = $this->storeManager->getStore()->getUrl(
```
Comment (sprankhub) [inline Helper/Attachment.php]: Why did you change this?
